src/OptionsPopup.qml

```diff
@@ -826,6 +846,20 @@ Window {
 
             addCmdline("cfg80211.ieee80211_regdom="+fieldWifiCountry.editText)
         }
+        if (chkUSBEther.checked) {
+            // keep parity with cli.cpp
```

Comment:
This "keep parity with..." feels a bit fragile? It's probably something that you should discuss with @tdewey-rpi  , but I wonder if it might be worth extracting out a common function that `OptionsPopup.qml` and `cli.cpp` could both call?

Reply:
That would be ideal, if we were operating across a single language - however, we are not - and that means any shared class between OptionsPopup and cli.cpp will have to be bound into javascript types and the QML engine - a lot of change for a small duplicated section.

Reply:
Fair enough.

Reply:
_(Putting it into a resource or adding these lines later in the process before writing the files (if the bool is true) may be a possible option...)_